### PR TITLE
Mark autotrack field as deprecated.

### PR DESCRIPTION
**Pull request by cmihail**
New clients should use the new auto tracking mode instead of autotrack field, and
in case they want to disable autotrack they should contact us directly
as we prefer having this option in our DB instead of GTM.

Tested:
* imported template without changes into my own GTM
* created tag from template
* change autotrack to false from default true
* modified template so that autotrack is marked as deprecated
* verified that autotrack is still false even with deprecation message

### CODE REVIEW

**Comment by cmihail** [comment]
I'm also discussing this topic with Swapnil directly, i.e. if the new text looks good. Once I get his approval I'll mention it here.

**Review by smnkrt — APPROVED** [review]
I am not sure how/if average folk understands `deprecated`, IMO looks good, but let's wait for Swapnil. 
We should make sure to inform Support/CS people about this. And possibly update help center - for example [this page](https://help.leadfeeder.com/en/articles/3727657-how-to-install-leadfeeder-tracker-using-google-tag-manager-and-the-leadfeeder-tracker-template) mentioned the auto tracking in step 3

**Comment by cmihail** [comment]
> I am not sure how/if average folk understands `deprecated`, IMO looks good, but let's wait for Swapnil. We should make sure to inform Support/CS people about this. And possibly update help center - for example [this page](https://help.leadfeeder.com/en/articles/3727657-how-to-install-leadfeeder-tracker-using-google-tag-manager-and-the-leadfeeder-tracker-template) mentioned the auto tracking in step 3

I've asked Helena to update that page. Preferably no new client should use this setting anymore.

**Pull request closed without merging**
